src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,27 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
-        return null;
+        Comparator<Employee> employeeComparator = new Comparator<Employee>() {
+            @Override
+            public int compare(Employee o1, Employee o2) {
+                int ageCompared = Integer.compare(o1.getAge(), o2.getAge());
+
+                if (ageCompared != 0) {
+                    return ageCompared;
+                }
+
+                return o1.getName().compareTo(o2.getName());
+            }
+        };
```

Comment:
You can use a lambda expression to simplify the instantiation of the Comparator. This will make the code more concise and easier to read.